### PR TITLE
Making usingMultipleWriteLocations by default to true

### LINKED ISSUE #7848: Making usingMultipleWriteLocations by default to true
Making usingMultipleWriteLocations by default to true
This is same as https://github.com/Azure/azure-cosmosdb-java/issues/202 opened in V3

### PR DESCRIPTION
**Pull request by simplynaveen20**
Making usingMultipleWriteLocations by default to true.

closes #7848

### CODE REVIEW

**Comment by simplynaveen20** [comment]
> Please update the `getUsingMultipleWriteLocations()` API documentation. It still says that default value is False: https://github.com/Azure/azure-sdk-for-java/pull/7847/files#diff-407404062107740a4d57778b0d4e32abR234

done

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ConnectionPolicy.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
     private RetryOptions retryOptions;
     private boolean enableEndpointDiscovery = true;
     private List<String> preferredLocations;
-    private boolean usingMultipleWriteLocations;
+    private boolean usingMultipleWriteLocations = true;
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ConnectionPolicy.java]: It should not be ported to V3. Please track it.

Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ConnectionPolicy.java]: Ok

**Pull request merged**